packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -2,6 +2,21 @@ import * as React from 'react';
 import { IRectangle, IRenderFunction } from '../../Utilities';
 import { List } from './List';
 
+export const enum ScrollToMode {
+  /**
+   * Does not make any consideration to where in the viewport the item should align to.
+   */
+  auto,
+  /**
+   * Attempts to scroll the list so the top of the desired item is aligned with the top of the viewport.
+   */
+  top,
+  /**
```

Comment:
Just curious, why wouldn't you just always want the behavior to be:

Scroll the item to the center of the viewport?

Reply:
Actually in reading the issue and the react-virtualized approach, I'm fine with the change, but it seems that center is missing...

Reply:
Ok, I can add center.

Reply:
Done.